pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -4550,6 +4550,39 @@ The paths themselves are given in the direction of the first geometry.`,
 	//
 	// Transformations
 	//
+	"st_asmvtgeom": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "geometry", Typ: types.Geometry},
```

Comment:
since we don't have user defined arguments, you mind creating each of these as overloads:

st_asmvtgeom(geometry, bbox)
st_asmvtgeom(geometry, bbox,extent)
st_asmvtgeom(geometry, bbox,extent,buffer)

Reply:
Sure, added